### PR TITLE
css-media-queries - Yuliia P.

### PR DESCRIPTION
**Pull request by yuliia-p**
https://user-images.githubusercontent.com/106217355/178904134-09186e21-156f-437d-b454-f03e03771c53.mp4

### CODE REVIEW

**Comment by yuliia-p** [comment]
https://user-images.githubusercontent.com/106217355/178904174-1aa4ca7a-91fb-45ce-8731-f5ebbf09df8b.mp4

**Comment by yuliia-p** [comment]
https://user-images.githubusercontent.com/106217355/178904219-687a0ddb-b3c1-4ddc-a377-07a0d6c6bb04.mov

File: css-media-queries/at-media.html [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+      min-height: 100vh;
+      background-color: palevioletred;
+    }
+
+    /* your code here 👇 */
+@media only screen and (min-width: 768px) {
+body {
+  background-color: papayawhip;
+}
+}
```
Comment (codingcodymiller) [inline css-media-queries/at-media.html:20]: Careful with indentation 😬

File: css-media-queries/breakpoints.html [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+  @media only screen and (min-width: 575px) {
+      .card-wrapper {
+      flex-basis: 50%;
+      }
+    }
+   @media only screen and (min-width: 767px) {
+    .card-wrapper {
+      flex-basis: calc(100% / 3);
+    }
+    }
```
Comment (codingcodymiller) [inline css-media-queries/breakpoints.html:31]: Careful with indentation 😬

File: css-media-queries/meta-viewport.html [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+      width: 200px;
+      height: 200px;
+      background-color: blue;
+    }
+
+  @media only screen and (min-width: 568px) {
+      .box {
+        background-color: red;
+      }
+    }
```
Comment (codingcodymiller) [inline css-media-queries/meta-viewport.html:29]: Careful with indentation 😬

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

File: css-media-queries/at-media.html [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+    body {
+      min-height: 100vh;
+      background-color: palevioletred;
+    }
+
+    /* your code here 👇 */
+    @media only screen and (min-width: 768px) {
+    body {
+    background-color: papayawhip;
+      }
```
Comment (codingcodymiller) [inline css-media-queries/at-media.html:19]: This is still incorrect indentation

File: css-media-queries/breakpoints.html [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+    .card-wrapper {
+      padding: 0.5rem;
+      flex-basis: 100%;
+    }
+
+    /* your code here 👇 */
+     @media only screen and (min-width: 575px) {
+      .card-wrapper {
+      flex-basis: 50%;
+      }
```
Comment (codingcodymiller) [inline css-media-queries/breakpoints.html:25]: This is still incorrect indentation

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

**Comment by yuliia-p** [comment]
Screen Shot 2022-07-15 at 4 34 53 PM
Screen Shot 2022-07-15 at 4 35 05 PM
Screen Shot 2022-07-15 at 4 35 11 PM

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
This is still incorrect, properties are supposed to be indented by one tab from the selector

**Comment by yuliia-p** [comment]
Screen Shot 2022-07-18 at 9 20 28 PM

**Review by codingcodymiller — APPROVED** [review]

**Pull request merged**